apps/api/src/app/step-schemas/dtos/step-schema.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { Schema } from '@novu/framework';
+
+export type StepSchemaDto = {
+  controls: Schema;
```

Comment:
Maybe we could import JsonSchema directly instead of using this wrapper. Our framework doesn't seem to provide any unique functionality here, so there's no need for this extra dependency. Also, I think the DTO should be placed in the shared directory to make it accessible for the front-end client.

Reply:
totally agree i was my mismatch, thanks! 🙏

Reply:
i wanted to make it compatible with framework but we definitely can not over-engineer it at the moment.